sympy/core/expr.py

```diff
@@ -445,6 +445,11 @@ def is_constant(self, *wrt, **flags):
         all derivatives are zero then self is constant with respect to the
         given symbols.
 
+        3) finding out zeros of denominator expression with free_symbols and
+        multiplicative inverse for this expression. It won't be constant
+        if there are zeros. It gives more negative answerts for non-constants
```

Comment:
typo: answerts
suggest: for non-constants expression -> for expression that are not constant